javascript/chronometer.js

```diff
@@ -1,34 +1,44 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.setIntervalId = setInterval(() => {
+      this.currentTime++
+
+
+    }, 1000)
   }
 
   getMinutes() {
-    // ... your code goes here
+   return Math.floor(this.currentTime / 60)
+
   }
 
   getSeconds() {
-    // ... your code goes here
+    return this.currentTime % 60;
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if (value < 10){                // return (value < 10) ? "0" + value : value + "";
+      return "0" + value;
+    } else {
+      return value + "";
+    }
   }
 
   stop() {
-    // ... your code goes here
+    clearInterval(this.setIntervalId);
   }
 
   reset() {
-    // ... your code goes here
+    this.currentTime = 0;
   }
 
   split() {
-    // ... your code goes here
+   return this.computeTwoDigitNumber(this.getMinutes()) + ":" + this.computeTwoDigitNumber(this.getSeconds());
   }
 }
```

Comment:
Die Aufgaben sind so kurz, dass es übersichtlich genug ist. Um den Code noch übersichtlicher zu gestalten würde ich dir jedoch mindestens in dieser split Funktion empfehlen, Ergebnisse in Variablen abzuspeichern. Also z. B. so:
```
 split() {
    let minutes = this.computeTwoDigitNumber(this.getMinutes());
    let seconds = this.computeTwoDigitNumber(this.getSeconds());

    return `${minutes}:${seconds}`;
  }
```